procurement_groupby_analytic/README.rst

```diff
@@ -0,0 +1,55 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Procurement Group by Analytic
+===========
+
+This module group purchase order by analytic.
+When purchase order is created from procurement order set analytic on purchase
+order line.
+
+Installation
+============
+
+To install this module, you need to:
+
+* Click on install button
+
+Usage
+=====
+
+For further information, please visit:
+
+* https://www.odoo.com/forum/help-1
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/purchase-workflow/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/OCA/purchase-workflow/issues/new?body=module:%procurement_groupby_analytic%0Aversion:%200.1%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```

Comment:
No, because module version numbering is a totally mess. It's for recognizing which of the branches is the affected one. It is supposed that you're already running last available version (we should talked here better about revisions) of the module in the branch, and if not, the first suggestion made is that.